frontend/src/Components/Header.js

```diff
@@ -18,21 +37,33 @@ function Header() {
           {" "}
           <Search />{" "}
         </Searchdiv>
-        <h1 className="App-h1">The Bananalyst</h1>
+        <h1 className="App-h1">
+          <StyledLink to="/">The Bananalyst</StyledLink>
+        </h1>
         <nav className="App-nav">
           <h2>
             <div className="dropdown">
-              <button className="dropbtn">Movies</button>
+              {/* //Changed here: */}
+              <button name="movies-btn" className="dropbtn">
+                <StyledLink to="movie"> Movies </StyledLink>
+              </button>
               <div className="dropcont">
+                {/* TODO: Finalize navigation on all buttons. Currently on temporary  */}
                 <p>Popular Movies</p>
-                <p>Suggested Movies</p>
+                <p>
+                  <StyledLink to="movie/3">Suggested Movies</StyledLink>
```

Comment:
there's a movie/recommend route in app.js. Probably should change it to movie/suggested since the header has it as suggested